CHANGELOG.md

```diff
@@ -79,6 +79,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Pretty printing of buffers is now more robust. Instead of throwing an `internal error`,
   we now try best to print everything we can, and print an appropriate error message
   instead of crashing.
+- During 0xd07e92cd9 we accidentally removed adding already proven Prop sets to
```

Comment:
No need for changelog entry, the cache was working correctly at the last release.

Reply:
Deleted it :) Sorry.